libs/stache/src/lib/modules/action-buttons/action-buttons.component.spec.ts

```diff
@@ -189,12 +190,4 @@ describe('StacheActionButtonsComponent', () => {
     fixture.detectChanges();
     expect(fixture.nativeElement.querySelector('sky-search')).toBeNull();
   });
-
-  // This will allow documentation writers to not worry about proper attribute binding.
-  it('should allow hiding search with string "false"', () => {
```

Comment:
This methodology no longer works with strict mode (converting string "false" to `false` from the template).